### PR TITLE
v2

### PR DESCRIPTION
**Pull request by stdavis**
Highlights include an upgrade to use http-proxy-middleware which solved a lot of issues including experience builder support.

- [x] https://github.com/agrc/gcp-terraform/pull/188 needs to be merged and applied to prod before this can be merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 A web service for proxying requests to the Esri print service switching out locked down quad-words for wide open ones.
 
+[!IMPORTANT]
+Looking for information on how to upgrade to v2? Head to [this doc](docs/v2-migration.md).
```
Comment (steveoh) [inline README.md:6]: **suggestion**: if you are wanting the fancy highlighting [notice](https://github.com/orgs/community/discussions/16925) here the syntax is a little off. 

```suggestion
> [!IMPORTANT]
> Looking for information on how to upgrade to v2? Head to [this doc](docs/v2-migration.md).
```

File: firestore.rules [hunk 2]
```diff
@@ -0,0 +1,7 @@
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read: if true;
```
Comment (steveoh) [inline firestore.rules:4]: **question**: is it ok that any document can be read by anyone?

**Review by steveoh — APPROVED** [review]

File: firestore.rules (hunk 2 above)
Comment (stdavis) [reply firestore.rules:4]: It's not a big deal. Although, we will have some emails in there at some point. I actually don't need anything to be able to read this data other than the cloud run service. Is there a better way to achieve this?

**Comment by stdavis** [comment]
@steveoh This is ready for another look when you have a minute.

**Review by steveoh — APPROVED** [review]

**Pull request merged**